react-frontend/src/components/Home/bcLivesteam.js

```diff
@@ -0,0 +1,129 @@
+import React, { useContext } from 'react';
+import dateFormat from 'dateformat';
+import { Col, Row } from 'react-flexbox-grid';
+import { PageContainer } from '../StyleComponents/pageContent';
+import { Heading } from '../StyleComponents/headings';
+import Query from '../Query';
+import { LinkExternalButton } from '../StyleComponents/htmlTags';
+
+import LIVESTREAM_QUERY from '../../queries/digitalBCLivestream/livestreams';
+import { AppConfigContext } from '../../providers/AppConfig';
+const youTubeLogo = require('../../images/icons/yt_logo_rgb_light.png').default;
+
+const BCLivestream = () => {
+  const strapiMediaUrl = useContext(AppConfigContext)['state'][
+    'strapiMediaUrl'
+  ];
+  return (
+    <Query query={LIVESTREAM_QUERY}>
+      {({ data: { digitalLivestreamContact, digitalBcLivestreams } }) => {
+        const imageSource =
+          strapiMediaUrl + digitalBcLivestreams[0]?.EventImage?.url;
+        // if no next event is provided in strapi, this section will not render
+        const now = new Date();
+        const nextEventTime = new Date(digitalBcLivestreams[0]?.BroadcastTime);
+        if (nextEventTime && nextEventTime > now) {
+          return (
+            <div style={{ backgroundColor: '#fff' }}>
+              <PageContainer
+                style={{ backgroundColor: '#fff', paddingTop: '40px' }}
+              >
+                <Row between="xs">
+                  <Col sm={12} md={3}>
+                    <Heading>#DigitalBC Livestream</Heading>
```

Comment:
To match wireframe, please: 

1. reduce the `line-height` on this `<Heading>` to `1.2`, and
2. centre the items in this column

![Screen Shot 2021-09-22 at 4 26 20 PM](https://user-images.githubusercontent.com/16946297/134434967-79740e23-c1bf-4d68-92c4-73ead1e02d84.png)